src/generator/www/js/main.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function(){
+
+ var $email =$('#email');
+ var $name  =$('#name'); 
+ var $url  =$('#url');
+
+  $('#btn').click(function() { 
+  	
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+  var file_data = $('input[type="file"]')[0].files;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'file_data' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l11)